x-pack/plugins/apm/common/agent_configuration/setting_definitions/general_settings.ts

```diff
@@ -184,6 +331,44 @@ export const generalSettings: RawSettingDefinition[] = [
     includeAgents: ['java', 'dotnet', 'go'],
   },
 
+  {
+    key: 'trace_continuation_strategy',
+    validation: traceContinuationStrategyRt,
+    type: 'select',
+    defaultValue: 'continue',
+    label: i18n.translate(
+      'xpack.apm.agentConfig.traceContinuationStrategy.label',
+      {
+        defaultMessage: 'Trace continuation strategy',
+      }
+    ),
+    description: i18n.translate(
+      'xpack.apm.agentConfig.traceContinuationStrategy.description',
+      {
+        defaultMessage:
+          'This option allows some control over how the APM agent handles W3C trace-context headers on incoming requests. By default, the `traceparent` and `tracestate` headers are used per W3C spec for distributed tracing. However, in certain cases it can be helpful to not use the incoming `traceparent` header. Some example use cases:\n' +
+          '\n' +
+          '* An Elastic-monitored service is receiving requests with `traceparent` headers from unmonitored services.\n' +
+          '* An Elastic-monitored service is publicly exposed, and does not want tracing data (trace-ids, sampling decisions) to possibly be spoofed by user requests.\n' +
+          '\n' +
+          'Valid values are:\n' +
+          "* 'continue': The default behavior. An incoming `traceparent` value is used to continue the trace and determine the sampling decision.\n" +
+          "* 'restart': Always ignores the `traceparent` header of incoming requests. A new trace-id will be generated and the sampling decision will be made based on transaction_sample_rate. A span link will be made to the incoming `traceparent`.\n" +
+          "* 'restart_external': If an incoming request includes the `es` vendor flag in `tracestate`, then any `traceparent` will be considered internal and will be handled as described for 'continue' above. Otherwise, any `traceparent` is considered external and will be handled as described for 'restart' above.\n" +
+          '\n' +
+          'Starting with Elastic Observability 8.2, span links are visible in trace views.\n' +
+          '\n' +
+          'This option is case-insensitive.',
+      }
+    ),
+    options: [
+      { text: 'continue', value: 'continue' },
+      { text: 'restart', value: 'restart' },
+      { text: 'restart_external', value: 'restart_external' },
```

Comment:
You'll need i18n for the `text` al well.

Reply:
I think it's fine if the entries in the dropdown have the exact values as in the agent configuration file. That's what we do for other options, such as `capture_body` and `log_level`, too.

Reply:
Keeping the original option names untranslated sgtm